ethportal-api/Cargo.toml

```diff
@@ -11,10 +11,9 @@ categories = ["cryptography::cryptocurrencies"]
 authors = ["https://github.com/ethereum/trin/graphs/contributors"]
 
 [dependencies]
-alloy-consensus.workspace = true
-alloy-primitives.workspace = true
+alloy = { workspace = true, features = ["consensus", "rlp", "rpc-types", "serde"] }
 alloy-rlp.workspace = true
-alloy-rpc-types.workspace = true
+alloy-rpc-types-eth = "0.4.2"
```

Comment:
Huh, so are the `eth_*` types included by default in the `alloy-rpc-types` crate, but not included in the `rpc-types` feature?

Reply:
It's about "serde" feature of "alloy-rpc-types-eth" crate, which isn't enabled by following features: "rpc-types", "serde" (and I believe it should). This line solves it because that feature is enabled by default (but when imported via "alloy", they don't enable default features).